README.md

```diff
@@ -43,6 +43,7 @@ This is list of known compatible USB hubs:
 | Dell               | P2416D 24" QHD Monitor ([note](https://git.io/JUAu8))| 4     | 2.0 |           | 2017    |      |
 | Dell               | UltraSharp 1704FPT 17" LCD Monitor                   | 4     | 2.0 |`0424:A700`| 2005    | 2015 |
 | Dell               | UltraSharp U2415 24" LCD Monitor                     | 5     | 3.0 |           | 2014    |      |
+| Dell               | S2719DGF 27" WQHD Gaming-Monitor                     | 5     | 3.1 |           | 2018    |      |
```

Comment:
Can you please sort this line in table alphabetically - it should come before UltraSharps. Fill in `vid:pid` too :-). Thanks!